conda/dev.yml

```diff
@@ -5,30 +5,30 @@ channels:
 dependencies:
   # Base
   # =================
+  - python=3.9.13
+  - pip=22.2.2
   - configobj=5.0.6
   - jinja2=3.1.2
-  - mache >=1.3.2
-  - pillow=8.1.2
-  - pip=21.0.1
-  - python=3.7.10
+  - mache>=1.5.0
+  - pillow=9.2.0
   # Developer Tools
   # =================
-  # If versions are updated, also update 'rev' in `.pre-commit.config.yaml`
-  - black=22.3.0        # version from https://anaconda.org/conda-forge/black
-  - flake8=3.9.2        # version from https://anaconda.org/conda-forge/flake8
+  # If versions are updated, also update 'rev' in `.pre-commit-config.yaml`
+  - black=22.8.0        # version from https://anaconda.org/conda-forge/black
+  - flake8=5.0.4        # version from https://anaconda.org/conda-forge/flake8
   # This line also implicitly installs isort
-  - flake8-isort=4.0.0  # version from https://anaconda.org/conda-forge/flake8-isort
-  - mypy=0.910          # version from https://anaconda.org/conda-forge/mypy
-  - pre-commit=2.13.0   # version from https://anaconda.org/conda-forge/pre-commit
+  - flake8-isort=4.2.0  # version from https://anaconda.org/conda-forge/flake8-isort
+  - mypy=0.982          # version from https://anaconda.org/conda-forge/mypy
+  - pre-commit=2.20.0   # version from https://anaconda.org/conda-forge/pre-commit
+  - tbump=6.9.0
```

Comment:
`tbump` is now available on conda-forge so it doesn't need to be installed through `- pip :`